smriprep/cli/run.py

```diff
@@ -76,17 +75,17 @@ def get_parser():
 
     g_conf = parser.add_argument_group('Workflow configuration')
     g_conf.add_argument(
-        '--output-spaces', nargs='+', action=ParseTemplates,
-        help='paths or keywords prescribing standard spaces to which normalize the input T1w image'
-             ' (valid keywords are: %s).' % ', '.join('"%s"' % s for s in templates()))
+        '--output-spaces', nargs='+', type=Space.from_string,
```

Comment:
We can get rid of the Action.

One think `from_string` should probably do is to expand minimized entities (i.e., `res` -> `resolution` and `den` -> `density`).

Reply:
will this be a list of lists?

Reply:
what do you mean?

Reply:
`Space.from_string()` will return one or more `Space`s as a list. `nargs='+'` will compose a list of all arguments, so it seems like it will be a list of lists. I wonder why this is currently working

Reply:
Yeah, you're right.